data_safe_haven/resources/workspace/ansible/host_vars/localhost.yaml

```diff
@@ -31,15 +31,18 @@ package_categories:
     noble:
   - category: development
     common:
+      - aspnetcore-runtime-8.0
       - build-essential
       - cargo
       - cmake
       - default-jre
+      - dotnet-sdk-8.0
       - g++
       - gcc
       - gfortran
       - git-all
       - golang
+      - mono-devel
```

Comment:
Do we want dotnet AND mono? It's been a while since I had to think about this, but aren't they doing the same thing?

Reply:
I'm not familiar with them. We could be opinionated at say mono only, but both are available so it seems like a low cost. I'm not sure how compatible they are.

This is a bit different to Matlab/Octave where there would also be licensing problems.

Reply:
Mono was an open-source implementation of DotNet when it didn't support Linux. Subsequently, Mono was bought by Microsoft and DotNet core is available for Linux. I think we probably don't need Mono here.

Reply:
> IIRC Mono was an open-source implementation of DotNet when it didn't support Linux.

That might be true but I think there are still an important difference due to licenses and that .Net core doesn't implement all .Net functions (_e.g._ silverlight/moonlight).

I would suspect they are very similar but there would be some edge cases which might require one implementation in particular.
If we were to remove one, I would lean towards keeping on Mono as it is FOSS.

Reply:
Wasn't Silverlight deprecated in the 2010s?

No strong preference between the two - can you ask someone who actually uses a .NET language which is better?

Reply:
Can you point me in the direction of someone who uses .NET :laughing:? (I'm honestly not aware of anyone).

Reply:
@evelinag?

Reply:
Looks like .NET is the way to go. It is MIT now and more actively developed.

We can worry about edge cases where Mono is needed over .NET when they happen :+1: